py-polars/polars/dataframe/frame.py

```diff
@@ -9237,6 +9237,25 @@ def unique(
         This method will fail if there is a column of type `List` in the DataFrame or
         subset.
 
+        Notes
+        -----
+        If you're coming from pandas, then
+
+        .. code-block:: python
+
+            # polars
+            df.unique(["a", "b"])
+
+        is equivalent to
+
+        .. code-block:: python
+
+            # pandas
+            df.drop_duplicates(["a", "b"])
+
+        though note that the output ordering is not stable unless
```

Comment:
Maybe remove the "though" and just say "Note that..."? The "though" feels a bit awkward.